src/AmpSlugCustomizationWatcher.php

```diff
@@ -0,0 +1,139 @@
+<?php
+/**
+ * Class AmpSlugCustomizationWatcher.
+ *
+ * @package AmpProject\AmpWP
+ */
+
+namespace AmpProject\AmpWP;
+
+use AmpProject\AmpWP\Infrastructure\Registerable;
+use AmpProject\AmpWP\Infrastructure\Service;
+
+/**
+ * Service for redirecting mobile users to the AMP version of a page.
+ *
+ * @package AmpProject\AmpWP
+ */
+final class AmpSlugCustomizationWatcher implements Service, Registerable {
+
+	/**
+	 * The slug (query var) that was customized.
+	 *
+	 * @var string|null
+	 */
+	protected $customized_slug = null;
```

Comment:
Redundant `null` default value.

```suggestion
	protected $customized_slug;
```

Reply:
Obsolete as of 52efa8043a1e646e92a9223d95bfe1a0e99b27e3